src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -77,6 +77,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\WebJobs.Script.Abstractions\WebJobs.Script.Abstractions.csproj" />
```

Comment:
This is just temp right?

Reply:
`FunctionMetadata` is now part of the `WebJobs.Script.Abstractions` . Udpated FunctionMetadata to include new property `Retry`

Reply:
I mean the adding of the project reference is temporary, to allow you to easily test out your changes? You'll revert this before checking in, because we want a nuget package dependency, not a project dependency.

Reply:
Ahh. Thanks. Makes sense now :) Yes will fix the PR once webjobs SDK is on myget

Reply:
fixed references!